src/app/utils/chromeStorage.ts

```diff
@@ -19,35 +19,35 @@ class ChromeStorage {
     return runtimeMap.get(this).lastError;
   }
 
-  setItem(key: string, item: any): Promise<any> {
+  setItem(key: string, item: any): Promise<void> {
     return new Promise((resolve, reject) => {
       this.getDriver().set({ [key]: item }, () => {
         if (this.hasError()) {
           return reject(this.getError());
         }
-        return resolve(true);
+        return resolve();
       });
     });
   }
 
-  getItem<T = any>(key: string): Promise<T> {
+  getItem<T = any, D = undefined>(key: string, defaultValue?: D): Promise<T | D> {
     return new Promise((resolve, reject) => {
       this.getDriver().get(key, (response: any) => {
         if (this.hasError()) {
           return reject(this.getError());
         }
-        return resolve(response[key]);
+        return resolve(response[key] ?? defaultValue);
       });
     });
   }
 
-  removeItem(key: string): Promise<any> {
+  removeItem(key: string): Promise<void> {
```

Comment:
I tried changing this and the setItem one to boolean, but the seedVault stuff started to complain. Checking the rest of the code, nothing was actually using the return value, so I removed it.